src/ResourceManager/Websites/Commands.Websites/Az.Websites.psd1

```diff
@@ -95,7 +95,8 @@ CmdletsToExport = 'Get-AzAppServicePlan', 'Set-AzAppServicePlan',
     'New-AzWebApp', 'Remove-AzWebAppBackup', 
     'Reset-AzWebAppPublishingProfile', 'Restart-AzWebApp', 
     'Set-AzWebApp', 'Start-AzWebApp', 'Stop-AzWebApp', 
-    'Get-AzWebAppSnapshot', 'Restore-AzWebAppSnapshot'
+    'Get-AzWebAppSnapshot', 'Restore-AzWebAppSnapshot',
+    'Get-AzDeletedWebApp'
```

Comment:
Missing one of the cmdlets

Reply:
Please follow these instructions to create the right aliases for the Az -> AzureRm netcore cmdlets. https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/create-alias-mappings.md